x/staking/types/params.go

```diff
@@ -32,10 +32,10 @@ var (
 	KeyBondDenom     = []byte("BondDenom")
 )
 
-var _ params.ParamSet = (*Params)(nil)
```

Comment:
why a new type?

Reply:
swaggo cant see the different between the same type names.

they get pulled into the swagger file and they all show up as types.Params and then the definitions are wrong and always out of date